internal/pkg/config/env.go

```diff
@@ -28,18 +28,20 @@ type Environment struct {
 
 // CustomizeEnv represents the custom environment config.
 type CustomizeEnv struct {
-	ImportVPC *ImportVPC `json:"importVPC,omitempty"`
-	VPCConfig *AdjustVPC `json:"adjustVPC,omitempty"`
+	ImportVPC      *ImportVPC `json:"importVPC,omitempty"`
+	VPCConfig      *AdjustVPC `json:"adjustVPC,omitempty"`
+	ImportCertARNs []string   `json:"importCertARNs,omitempty"`
 }
 
 // NewCustomizeEnv returns a new CustomizeEnv struct.
-func NewCustomizeEnv(importVPC *ImportVPC, adjustVPC *AdjustVPC) *CustomizeEnv {
-	if importVPC == nil && adjustVPC == nil {
+func NewCustomizeEnv(input CustomizeEnv) *CustomizeEnv {
```

Comment:
nit: do we even need this function :P it doesn't really seem to provide much

Reply:
I think it still adds value so that if any of those fields are not set we won't write to the param store instead of an empty struct. no?